data/transactions/logic/eval.go

```diff
@@ -2526,6 +2527,164 @@ func opEd25519verify(cx *EvalContext) {
 	cx.stack = cx.stack[:prev]
 }
 
+// leadingZeros needs to be replaced by big.Int.FillBytes
+func leadingZeros(size int, b *big.Int) ([]byte, error) {
+	data := b.Bytes()
+	if size < len(data) {
+		return nil, fmt.Errorf("insufficient buffer size: %d < %d", size, len(data))
+	}
+	if size == len(data) {
+		return data, nil
+	}
+
+	buf := make([]byte, size)
+	copy(buf[size-len(data):], data)
+	return buf, nil
+}
+
+func opEcDsaVerify(cx *EvalContext) {
```

Comment:
Why capital D? I don't actually know what the letters stand for but it seems like if this should be called EcDsa, that would imply the opcode should be ec_dsa.  I think I have normally seen them all run together, so I would only capitalize E here.

Reply:
I agree capital D is a bit odd